main.go

```diff
@@ -166,26 +168,23 @@ func main() {
 		})
 	}
 
+	tlsConfig := opts.GetTLSConfigPath()
```

Comment:
Shouldn't this suffice?
```suggestion
	tlsConfig := opts.TLSConfig
```